k8s/controllers/solution/solution_controller.go

```diff
@@ -37,10 +40,45 @@ type SolutionReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *SolutionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	log := ctrllog.FromContext(ctx)
+	log.Info("Reconcile Solution " + req.Name + " in namespace " + req.Namespace)
 
 	// TODO(user): your logic here
 
+	solution := &solutionv1.Solution{}
```

Comment:
We don't define async type in metadata.json. If you want k8sbridge to respect reconcile logic, you need to generate the right provisioning status.